test/e2e/pvutil.go

```diff
@@ -62,19 +62,6 @@ type persistentVolumeConfig struct {
 	namePrefix string
 }
 
-// Delete the nfs-server pod. Only done once per KubeDescription().
-func nfsServerPodCleanup(c clientset.Interface, config VolumeTestConfig) {
```

Comment:
As it's written now, NFS has been a way to provide a hostPath volume.  Provider specific tests (e.g. GCE)  define PVs that use the vendor's PersistentVolumeSource.  So I was thinking that it's only relevant to our NFS & disruptive suite.